data/operators/office/government.json

```diff
@@ -854,9 +854,22 @@
         "operator:wikidata": "Q4925621"
       }
     },
+    {
+      "displayName": "Jabatan Imigresen Malaysia",
+      "locationSet": {"include": ["my"]},
+      "matchNames": ["imigresen"],
+      "tags": {
+        "government": "immigration",
+        "office": "government",  
+        "operator": "Jabatan Imigresen Malaysia",
+        "operator:short": "JIM",
```

Comment:
Generally speaking, immigration bureaus only exist in cities with customs ports? Approximately how many JIM are there across Malaysia and how many have been mapped in OSM? (Does it satisfy notability)

Reply:
In Malaysia, JIM are responsible for things like renewing passport and renewing worker pass/visa.

According to the official website
(https://www.imi.gov.my/index.php/en/contact-us/adress-location/), there are around 150 location. This include their branch, detention depot and entrance point.

Right now only 9 location have been mapped.
http://overpass-turbo.eu/s/1GcK

Reply:
Sorry, I don’t know enough about the local situation. If there are so many theoretically, it is indeed worthy of being added. Thank you for talking about this!